### PR TITLE
Updated example requests

### PR DESCRIPTION
**Pull request by v-stepanov**
@rcillo @antban @jkliff please review

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/zalando/nakadi/pull/219" target="_blank">ReviewNinja

File: README.md [hunk 1]
```diff
@@ -74,7 +74,8 @@ curl --request POST \
   "name": "order.ORDER_RECEIVED",
   "owning_application": "order-service",
   "category": "business",
-  "partition_key_fields": [],
+  "partition_strategy": "random",
+  "enrichment_strategies": ["METADATA_ENRICHMENT"],
```
Comment (rcillo) [inline README.md]: The enrichment strategy name should be lowercase.

Comment (v-stepanov) [inline README.md]: done

**Comment by rcillo** [comment]
:shipit:

**Pull request merged**